tests/test_repoquery.py

```diff
@@ -141,3 +145,20 @@ def test_rpm2py_format(self):
         fmt = dnf.cli.commands.repoquery.rpm2py_format(
             '%{name}-%{repoid} :: %-40{arch}')
         self.assertEqual(fmt, '{0.name}-{0.repoid} :: {0.arch:>40}')
+
+
+class TestObsoletes(unittest.TestCase):
+    @support.mock.patch('sys.stdout', new_callable=io.BytesIO)
+    def test_obsoletes(self, mock):
+        exp_list = [
+            u'python-elementtree',
+            u'python-ctypes',
+            u'python-sqlite',
+            u'python-uuid',
+            u'python-hashlib',
+            u'python2']
+        main.user_main(["repoquery", "--obsoletes", "python"],
```

Comment:
Sorry, this is probably my mistake, I don't understand. What resources you are talking about? How should they be updated?